text/0083-component.md

```diff
@@ -0,0 +1,118 @@
+|   |  Tracer(Component)       |   |  Tracer(Component)       |   |
+|   |  Meter(Component)        |   |  Meter(Component)        |   |
+|   |                          |   |                          |   |
+|   +--------------------------+   +--------------------------+   |
+|                                                                 |
+|      Instrumented Library 3         Instrumented Library 3      |
+|   +--------------------------+   +--------------------------+   |
+|   |  Tracer(Component)       |   |  Tracer(Component)       |   |
+|   |  Meter(Component)        |   |  Meter(Component)        |   |
+|   |                          |   |                          |   |
+|   +--------------------------+   +--------------------------+   |
+|                                                                 |
++-----------------------------------------------------------------+
+```
+
+Every application has one `TracerProvider` and one `MeterProvider` which have a
+`Resource` associated with them that describes the Application.
+
+Every instrumented library has one `Tracer` and one `Meter` which have a
+`Component` associated with them that describes the instrumented library.
+
+In case of multi-application deployments like Spring boot (or old Java
+Application servers) every Application will have it's own `TracerProvider` and
+`MeterProvider` instances.
+
+## Internal details
+
+This change affects only the OpenTelemtry protocol, and propose a way to
+represent the telemetry data in a structural way, here is an example for
+metrics:
+
+```proto
+// Resource information.
+message Resource {
+  // Set of labels that describe the resource.
+  repeated opentelemetry.proto.common.v1.AttributeKeyValue attributes = 1;
+}
+
+// Component is a message representing the instrumenting library informations
+// such as the fully qualified name and version.
+message Component {
+  string name = 1;
+  string version = 2;
+  // Can be extended with attributes.
+}
+
+// A collection of ComponentMetrics from a Resource.
+message ResourceMetrics {
+  // The Resource for the metrics in this message.
+  // If this field is not set then no Resource is known.
+  Resource resource = 1;
+
+  // A list of metrics that originate from a Resource.
+  repeated ComponentMetrics component_metrics = 2;
+}
+
+// A collection of Metrics from a Component.
+message ComponentMetrics {
+  // The Component information for the metrics in this message.
+  // If this field is not set then no Component is known.
+  Component component = 1;
+
+  // A list of metrics that originate from a Component.
+  repeated Metric metrics = 2;
+}
+```
+
+## Trade-offs and mitigations
+
+Adding a new concept into the OpenTelemetry protocol and the exporter framework
+may be overkill, but this concept maps easily to an already existing concept
+in the API, and it is easy to explain to the users.
+
+## Prior art and alternatives
+
+An alternative approach was proposed in the proto [PR comment](
```

Comment:
Apart from renaming LibraryInfo to Component, this proposal is the same, correct?